ansible/roles/grav/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: "Install dependencies"
+  apt: name={{ item }} update_cache=yes state=latest
+  with_items:
+    - 'unzip'
+    - 'openssl'
+    - 'zip'
+
+# Environment
+- name: create grav group
+  group:
+    name: "{{ grav_group }}"
+    state: present
+
+- name: create grav user
+  user:
+    name: "{{ grav_user }}"
+    group: "{{ grav_group }}"
+    uid: "{{ grav_uid }}"
+    createhome: no
+    shell: /sbin/nologin
+    comment: "{{ grav_user_comment }}"
+    state: present
+
+- name: Create grav install folder
+  file:
+    path: "{{ item }}"
+    owner: "{{ grav_user }}"
+    group: "{{ grav_group }}"
+    state: directory
+  with_items:
+  - "{{ grav_path }}"
+  - "{{ grav_tmp_path }}"
+  - "{{ grav_install_path }}"
+
+- include: setup-php.yml
+- include: setup-nginx.yml
+- include: setup-grav.yml
```

Comment:
What's the reason to use some sort of PHP CMS instead of bare static website generator? As far as I see, it's used just to display text to user.